frontend/packages/console-shared/src/components/drawer/__tests__/Drawer.spec.tsx

```diff
@@ -85,13 +85,14 @@ describe('DrawerComponent', () => {
   });
 
   it('should handle resizing', () => {
+    const e = { preventDefault: jest.fn() };
     const data = {} as DraggableData;
     const onChange = jest.fn();
     const wrapper = shallow(<Drawer resizable defaultHeight={100} onChange={onChange} />);
     wrapper
       .find(DraggableCoreIFrameFix)
       .props()
-      .onStart({ pageY: 500 } as any, data);
+      .onStart({ pageY: 500, ...e } as any, data);
```

Comment:
This seems like just associating a variable with extra steps 😄 

```
// const e = { preventDefault: jest.fn() };
const preventDefault = jest.fn();
```

```suggestion
      .onStart({ pageY: 500, preventDefault } as any, data);
```

Reply:
Updated this